test/acceptance/features/steps/steps.py

```diff
@@ -466,3 +468,10 @@ def validate_secret_empty(context, secret_name):
                       ignore_exceptions=(json.JSONDecodeError,))
     except polling2.TimeoutException:
         pass
+
+
+@given(u'"{image}" image is deployed as "{deployment_name}"')
```

Comment:
IMHO, it reads better if we declare the step as:

"{app_name} is deployed from image {image_ref}"

Reply:
okay.